psychopy_eyetracker_sr_research/sr_research/eyelink/eyetracker.py

```diff
@@ -289,6 +294,7 @@ def setConnectionState(self, enable):
                         self._eyelink.open(host_pc_ip_address)
                         pylink.flushGetkeyQueue()
                         self._eyelink.setOfflineMode()
+                        self._initConnection()
```

Comment:
this now attempts to configure the eyelink session for each time the connection is opened